packages/cli/src/commands/deploy/index.ts

```diff
@@ -475,7 +493,12 @@ export default async (client: Client) => {
 
     if (!localConfig.builds || localConfig.builds.length === 0) {
       // Only add projectSettings for zero config deployments
-      createArgs.projectSettings = { sourceFilesOutsideRootDirectory };
+      createArgs.projectSettings =
+        status === 'not_linked'
```

Comment:
Do we need to handle `status === 'error'` here too?

Reply:
Unreachable I believe. There is a check higher up that is:

```js
  const link = await getLinkedProject(client, path);

  if (link.status === 'error') {
    return link.exitCode;
  }

  let { org, project, status } = link;
  ```